File: corpus/typescript/fixture_09.ts
type CartItem9 = { price: number; qty?: number };
export function calculateDiscount9(cartItems: CartItem9[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 90 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook9 {
  lookupPrice(sku: string): number { return sku.length + 9; }
}

File: corpus/python/fixture_09.py
def calculate_discount_9(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 90:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook9:
    def lookup_price(self, sku):
        return len(sku) + 9
